modules/visitedMenu.js

```diff
@@ -0,0 +1,18 @@
+const links = document.querySelectorAll('.links li');
+
+function resetLinks() {
+  links.forEach((li) => {
+    li.classList.remove('active');
+  });
+}
+
+function initVisitedMenu() {
+  links.forEach((li) => {
+    li.addEventListener('click', () => {
+      resetLinks();
+      li.classList.add('active');
+    });
+  });
+}
+/* eslint-disable import/prefer-default-export */
+export { initVisitedMenu };
```

Comment:
- [ ] Please don't disable linter errors instead fix them manually, It will help you to write more clean and professional code.